sw/device/tests/rv_core_ibex_address_translation_test.c

```diff
@@ -235,6 +233,58 @@ void check_ibus_map(dif_rv_core_ibex_t *ibex_core) {
   CHECK(illegal_instr_fault);
 }
 
+void check_dbus_map(dif_rv_core_ibex_t *ibex_core) {
+  CHECK(memcmp(make_lower_case, remapped_function, kRemapAlignment) != 0,
```

Comment:
I suggest using the macro `CHECK_ARRAYS_EQ`  `CHECK_ARRAYS_NE` as it will nicely log in case of error.